### DISCUSSION TITLE
Data Retrieval Limits and Update Frequency for Amazon Selling Partner API Reports

### DISCUSSION THREAD

**Discussion by Shoxyk** [amzn/selling-partner-api-models#3801] [General]
Hi everyone,

I'm reaching out to the community to inquire about the data retrieval limitations and update frequency for several Selling Partner API reports that I cannot find in the documentation. I'm particularly interested in the following reports:

- GET_FBA_REIMBURSEMENTS_DATA
- GET_FBA_FULFILLMENT_CUSTOMER_RETURNS_DATA
- GET_FBA_FULFILLMENT_CUSTOMER_SHIPMENT_REPLACEMENT_DATA
- GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2
- GET_LEDGER_DETAIL_VIEW_DATA
- GET_FBA_MYI_ALL_INVENTORY_DATA
- GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA

I'd appreciate any insights on the following:
1. **Maximum Data Returned per Call:** Is there a maximum number of days of data that can be retrieved in a single API call for each of these reports?
2. **Historical Data Availability:** How far back in time can I access data using these reports (e.g., 30 days, 90 days, or longer)?
3. **Data Update Frequency:** How often is the data in these reports updated by Amazon (e.g., daily, hourly, etc.)?

Any information or personal experiences shared by the community would be greatly helpful. Thanks in advance for your assistance!

**Comment by Javadebi** [comment]
no one is going to answer any question in here. good luck
